### PR TITLE
feat: 🇳🇴 Added Norwegian Bokmål

### PR DESCRIPTION
**Pull request by rubjo**
## Description
i18n: Added Norwegian Bokmål (nb)

### Checklist:

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you linted your code locally prior to submission?
* [x] Have you successfully ran app with your changes locally?

### CODE REVIEW

File: src/shared/locales/all.js [hunk 1]
```diff
@@ -132,6 +134,12 @@ const resources = {
       ...appLocalePl
     }
   },
+  'nb': {
```
Comment (agalwood) [inline src/shared/locales/all.js:137]: Please adjust the order of 'nb' to before 'pl' .

**Review by agalwood — CHANGES_REQUESTED** [review]
Please adjust the code : D

File: src/shared/locales/all.js (hunk 1 above)
Comment (rubjo) [reply src/shared/locales/all.js:131]: :)

**Review by agalwood — APPROVED** [review]

**Comment by agalwood** [comment]
Tks

**Pull request merged**